src/components/help-page/__tests__/help-page-test-cases.js

```diff
@@ -11,59 +10,78 @@ testCases.basic = {
   element: <Basic />
 };
 
-testCases.noThumbs = {
+testCases.tutorials = {
+  component: HelpPage,
+  description: 'Tutorials only, no heading',
+  props: {
+    introText: 'Our Help page contains tutorials to help you get started.',
+    data: [
+      {
+        pages: [
+          {
+            title: 'Example one',
+            description:
+              'Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+            path: 'path'
+          },
+          {
+            title: 'Example two',
+            description:
+              'Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+            path: 'path'
+          }
+        ]
+      }
+    ]
+  }
+};
+
+testCases.customIntroText = {
   component: HelpPage,
-  description: 'Cards with no images and full-width',
+  description: 'No intro text',
   props: {
-    frontMatter: {
-      title: 'Title',
-      description: 'Description.'
-    },
-    cardContainers: [
-      <CardContainer
-        title="Container title one"
-        path="#container-title-one"
-        fullWidthCards={true}
-        cards={[
-          <Card
-            key="0"
-            title="Example one"
-            path="path"
-            description="Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-          />,
-          <Card
-            key="1"
-            title="Example two"
-            path="path"
-            description="Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-          />
-        ]}
-      />,
-      <CardContainer
-        title="Container title two"
-        path="#container-title-two"
-        fullWidthCards={true}
-        cards={[
-          <Card
-            key="0"
-            title="Example one"
-            path="path"
-            description="Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-          />,
-          <Card
-            key="1"
-            title="Example two"
-            path="path"
-            description="Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-          />,
-          <Card
-            key="2"
-            title="Example two"
-            path="path"
-            description="Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-          />
-        ]}
-      />
+    introText: '',
```

Comment:
This seems like a tricky pattern... If `introText` is not included as a prop, then the default text is used and to not include any `introText` you have to include an empty string. I wonder if we should just get rid of the default and hardcode it for each page? What do you think?

Reply:
That makes sense, especially since we'll be splitting the Help pages into Tutorials and Troubleshooting -- I think this would have made sense if we were keeping Help pages in tact. I'll update to remove the prop all together.